new/components/rabbitmq/scripts/create.sh

```diff
@@ -4,92 +4,108 @@
 
 CONFIG_REL_PATH="components/rabbitmq/config"
 
-export ERLANG_SOURCE_URL=$(ctx node properties erlang_rpm_source_url)
-export RABBITMQ_SOURCE_URL=$(ctx node properties rabbitmq_rpm_source_url)
-export RABBITMQ_FD_LIMIT=$(ctx node properties rabbitmq_fd_limit)
+export RABBITMQ_USE_EXTERNAL=$(ctx -j node properties rabbitmq_use_external)
```

Comment:
That's fair enough- I was following the rabbit SSL paradigm but that's simple enough as there's only two places it's actually used (the rabbit scripts).